src/Debug.c

```diff
@@ -46,6 +47,36 @@ void Ros_Debug_BroadcastMsg(char* fmt, ...)
 
     if (ros_DebugSocket == -1)
         Ros_Debug_Init();
+    
+   // Timestamp
+
```

Comment:
```suggestion
```

Reply:
I'm sorry to have missed the removal of this blank line. I have made the changes and removed it.